docs(README.md): adds git config section (#381)
Shift3/standards-and-practices: README.md

- Please read _all_ the links included:
- Take a look at [our **Guidebook**](https://bwtc-handbook.notion.site/BWTC-Handbook-71256770d53f46439aecd989b07d1533) for all your (mostly) non-code related needs!

> #### GitHub

Reviewer: I almost feel like a whole other header for this little bit of information isn't neccesary. We could just add it under the heading for "New to BitWise"?

- Please read _all_ the links included:
- Take a look at [our **Guidebook**](https://bwtc-handbook.notion.site/BWTC-Handbook-71256770d53f46439aecd989b07d1533) for all your (mostly) non-code related needs!

### GitHub

> - Please create a Github account using your **work** email

Reviewer 2: Why do we want to make everyone use their work email for their GH usernames? I believe GH allows for multiple email addresses to be associated with any account that you own. Just want to see what the thought process is here.

Author: The thought was to only allow those currently employed to have access. It was mentioned to me to create a GH account with my work email, which meant that I had to edit my git config. As I was going through this process I thought that including our work email to be relevant. If there are no policies in place restricting personal accounts associated with work repositories, then I can remove line 27.

Reviewer 3: GitHub recommends using a single personal account, as mentioned in the links to the answer on this stackexchange answer: https://webapps.stackexchange.com/questions/76618/should-i-use-my-personal-github-account-for-work

I believe creating multiple GH accounts creates a level of personal management complexity which is unnecessary and doesn't bring any obvious benefits, plus it's not a use case encouraged or supported by GH itself. 

My recommendation is that we do not require employees to create a new account. Our GH teams and organizations already offer the necessary isolation and security. Now if someone *wants* to create a different account, I think that is fine. But as a requirement I strongly disagree.

It is still worth clarifying our expectations here since it sounds like contradicting instructions have been given out. Maybe here we can encourage employees to use their personal account and link to this official GH page which also mirrors the suggestions: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts

- Once an account has been created, open your terminal to configure your Git username/email.

>   <a href="https://support.atlassian.com/bitbucket-cloud/docs/configure-your-dvcs-username-for-commits/">![Gitflow diagram](./git-config.png)</a>

Reviewer 2: While `a` tags and html elements will definitely work within GH docs, lets keep that markdown standard for now. Its what we use for almost all of our other docs. I believe this should be equivalent: 

```suggestion
    Once an account has been created, open your terminal to configure your Git username/email. [Gitflow diagram](git-config.png)
  

```

Reviewer 2: Also how do you feel about making the GitHub/git config setup its own section within our documentation and removing the png? That way, we can stay relevant and not need to update the png if/when GH changes how they want us to set up our config.

Reviewer: @michaelachrisco I believe this is what you wanted

```suggestion
  [![Gitflow diagram](./git-config.png)</a>](https://support.atlassian.com/bitbucket-cloud/docs/configure-your-dvcs-username-for-commits/)
```

- Please read _all_ the links included:
- Take a look at [our **Guidebook**](https://bwtc-handbook.notion.site/BWTC-Handbook-71256770d53f46439aecd989b07d1533) for all your (mostly) non-code related needs!

> Get set up with [Github](/standards/git-config.md)

Reviewer 2: This is such a nitpick....but I think this should be **GitHub**
```suggestion
Get set up with [GitHub](/standards/git-config.md)
```

I don't think of this as a blocker, but it would be nice to stay consistent with company names. We probably have `Github` all over the S&P to be honest.

- [Commit Messages](/standards/commits.md) at Bitwise

#### Code Versioning

- [Bitwise SemVer](/standards/code-versioning.md)

#### Project Setup

> 

Reviewer 2: Thank you for making our markdown consistent (ie linting)! Looks much better looking at the source.
